docs/user-guide/src/irso/install-basics.md

```diff
@@ -0,0 +1,157 @@
+# Installing Ironic
+
+This document covers installing Ironic in different scenarios. You need to
+answer a few questions before you can pick the one that suits you:
+
+- Which physical network interface will be used for provisioning? Without any
+  configuration, Ironic will use the host cluster networking.
+
+- If you use a dedicated network interface, are you going to use the built-in
```

Comment:
A little bit confused about this: if we have multiple control plane nodes, should we make sure one interface on each control plane node has an IP address (which means one ironic per control plane node using that IP address)? Shouldn't that IP address be the same, or shouldn't we have a proxy?

Reply:
The role of keepalived is to maintain the same IP address, regardless of which node is hosting Ironic. I'll try to clarify a bit.

Reply:
So the bottom line is: Make sure you have a usable IP address on the control plane node hosting ironic. It's possible to use keepalived for this. 
Its clearer now, thank you!